src/formatter.js

```diff
@@ -230,6 +248,7 @@ let _formatters = {
 	'F|file' : _formatFile,
 	'K|map|MAP' : _formatMapMessage,
 	'L|line' : _formatLineNumber,
+	'column': _formatColumn,
```

Comment:
This is not a layout pattern specified in the log4j2 specification (which is what I'm trying to replicate), but I'm actually fine with it.